app/src/main/java/camp/nextstep/edu/github/MainActivity.kt

```diff
@@ -1,11 +1,41 @@
 package camp.nextstep.edu.github
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.lifecycleScope
+import camp.nextstep.edu.github.databinding.ActivityMainBinding
+import dagger.hilt.android.AndroidEntryPoint
 
+@AndroidEntryPoint
 class MainActivity : AppCompatActivity() {
+    private val mainViewModel: MainViewModel by viewModels()
+    private lateinit var binding: ActivityMainBinding
+    private lateinit var githubAdapter: GitHubRepositoryInfoAdapter
```

Comment:
Nit: lateinit이어야하는 이유가 특별하게 없을 때는 바로 선언해주는 것도 좋아요~
val로 선언할 수 있다면 불변객체의 장점을 가질 수 있기 때문입니다..!

```suggestion
    private val githubAdapter: GitHubRepositoryInfoAdapter = GitHubRepositoryInfoAdapter()
```